galicaster/mediapackage/mediapackage.py

```diff
@@ -1431,10 +1431,10 @@ def addSeriesDublincoreAsString(self, content, name=None, rewrite=False):
         assert path.isdir(self.getURI())
 
         f_path = path.join(self.getURI(), name if name != None else 'series.xml' ) #FIXME
-
         f = open(f_path, 'w')
         f.write(content)
         f.close()
+        f = open(f_path)
```

Comment:
Why is this file opened again? It's a local variable and it's never used again in this method.